docs/antora.yml

```diff
@@ -1,5 +1,11 @@
+#
+# File managed by ModuleSync - Do Not Edit
+#
+# The name and title can be customized in `.sync.yml` with `'docs/antora.yml'.title`, `'docs/antora.yml'.name`
+#
+
 name: argocd
-title: Argo CD
+title: argocd
```

Comment:
I guess this could stay as is? Would need to be updated in .sync.yml though

Reply:
Yep it should stay as is.